app/scripts/controllers/app-state.js

```diff
@@ -23,6 +23,7 @@ export default class AppStateController extends EventEmitter {
       timeoutMinutes: 0,
       connectedStatusPopoverHasBeenShown: true,
       defaultHomeActiveTabName: null,
+      dismissSeedBackUpReminder: false,
```

Comment:
This should probably be in the preferences controller rather than appState. I believe the distinction barrier between these two should be that if the user toggles/sets a setting that it goes in preferences controller. If the state is set just due to some interaction with the app then it goes into the App state.

Reply:
@brad-decker I was wondering the same thing. Unless @danjm feels strongly that this should live in appState (https://github.com/MetaMask/metamask-extension/issues/10660#issuecomment-827835784)?

Reply:
@adonesky1 Yes the preferences controller is correct in this case. My suggestion to put a flag in the appstate controller was made when we were thinking that there would be a dismiss button on the notification itself. Since making that comment, we decided to put the toggle in settings, and so I should have added an updated comment to suggest use of the preferences controller.

Reply:
@brad-decker I've moved this method to the preferences controller